R/utils-varnish.R

```diff
@@ -44,7 +44,7 @@ varnish_vars <- function() {
 #'
 #'  - `sidebar` This is generated from [create_sidebar()] and is the same in the
 #'    learner and instructor globals with the exception of the first element.
-#'  - `more` This is the "More" dorpdown menu, which is created via [create_resources_dropdown()].
```

Comment:
I mean, sometimes you get a dropdown, sometimes it's a dorpdown 😜

Reply:
(thank you)

Reply:
😂